crates/db_schema/src/utils.rs

```diff
@@ -118,6 +118,19 @@ pub fn naive_now() -> NaiveDateTime {
   chrono::prelude::Utc::now().naive_utc()
 }
 
+pub fn post_to_comment_sort_type(sort: SortType) -> CommentSortType {
+  match sort {
+    SortType::Active | SortType::Hot => CommentSortType::Hot,
+    SortType::New | SortType::NewComments | SortType::MostComments => CommentSortType::New,
+    SortType::Old => CommentSortType::Old,
+    SortType::TopDay
+    | SortType::TopAll
+    | SortType::TopWeek
+    | SortType::TopYear
+    | SortType::TopMonth => CommentSortType::Top,
+  }
```

Comment:
`use SortType::*`

Reply:
? The SortType is already imported at the top.

Reply:
With that use statement, you can just write `TopWeek` instead of `SortType::TopWeek` etc, making the code shorter and easier to read.